### PR TITLE
DOC-1327: RS CLI Utilities - rladmin recover page

### PR DESCRIPTION
**Pull request by cmilesb**
Staging link: https://docs.redis.com/staging/jira-doc-1327-actual/rs/references/cli-utilities/rladmin/recover/

Still waiting on a couple of things:
- I haven't been able to figure out how to get databases into recovery mode, so I haven't been able to run `rladmin recover` commands on the databases in my cluster. 
- Awaiting feedback from Alon or Amiram about the import from s3 feature - not sure if this is something we want to document or not.

### CODE REVIEW

**Comment by cmilesb** [comment]
I removed the "fake" examples and opened [DOC-1409](https://redislabs.atlassian.net/browse/DOC-1409) in our backlog to add real examples to rladmin recover all and rladmin recover db.

I also am still waiting on PM for the rladmin recover s3_import, but will bug them more once the PM retreat is over.

File: content/rs/references/cli-utilities/rladmin/recover.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+$ rladmin list
+```
+
+### Parameters
+
+None.
+
+### Returns
+
+Displays a list of all recoverable databases. If no databases are in reovery mode, prints `No recoverable databases found`.
```
Comment (rrelledge) [inline content/rs/references/cli-utilities/rladmin/recover.md:68]:
```suggestion
Displays a list of all recoverable databases. If no databases are in recovery mode, prints `No recoverable databases found`.
```

File: content/rs/references/cli-utilities/rladmin/recover.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+### Returns
+
+Returns `Completed successfully` if the database was recovered. Otherwise, returns an error.
+
+## `recover db`
+
+Recovers a specific database in recovery mode.
+
+```sh
+$ rladmin recover db { db<id> | <name> }
```
Comment (rrelledge) [inline content/rs/references/cli-utilities/rladmin/recover.md:39]:
```suggestion
rladmin recover db { db:<id> | <name> }
```

File: content/rs/references/cli-utilities/rladmin/recover.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+---
+
+`rladmin recover` recovers databases in recovery mode.
+
+## `recover all`
+
+Recovers all databases in recovery mode.
+
+```sh
+$ rladmin recover all
```
Comment (rrelledge) [inline content/rs/references/cli-utilities/rladmin/recover.md:20]:
```suggestion
rladmin recover all
```

File: content/rs/references/cli-utilities/rladmin/recover.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+### Returns
+
+Returns `Completed successfully` if the database was recovered. Otherwise, returns an error.
+
+## `recover list`
+
+Shows a list of all databases that are currently in recovery mode.
+
+```sh
+$ rladmin list
```
Comment (rrelledge) [inline content/rs/references/cli-utilities/rladmin/recover.md:59]:
```suggestion
rladmin list
```

File: content/rs/references/cli-utilities/rladmin/recover.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+description: Recovers databases in recovery mode.
+weight: $weight
+alwaysopen: false
+toc: "true"
+headerRange: "[1-2]"
+categories: ["RS"]
+aliases:
+---
+
+`rladmin recover` recovers databases in recovery mode.
```
Comment (rrelledge) [inline content/rs/references/cli-utilities/rladmin/recover.md:13]:
```suggestion
Recovers databases in recovery mode.
```

**Review by rrelledge — APPROVED** [review]
Looks good to me. I just left some suggestions for typo fixes.

**Pull request merged**